package.json

```diff
@@ -41,8 +41,8 @@
     "webpack": "^4.0.0 || ^5.0.0"
   },
   "dependencies": {
+    "camelcase": "^6.2.0",
```

Comment:
Can we avoid using extra module, no need new dependencies just for name

Reply:
@alexander-akait ok, done, inlined camelcase dep and removed unused code.